docs/application/native/guides/app-management/data-control.md

```diff
@@ -46,7 +46,7 @@ To enable your application to use the data control functionality:
 
 2. For the provider, in the Tizen Studio, double-click **tizen-manifest.xml**, and in the manifest editor, go to **Advanced > Data Control**, and click **+** to add the provider details. Add the **Read** and **Write** access rights to both **SQL** and **Map** types.
 
-    You can set the data access to trusted, to allow other applications that are signed with the same certificate to access the data. Additionally, you can also define privileges to restrict access for applications having certain predefined privileges.
```

Comment:
It is possible to use app-defined privileges also

Reply:
Thank you for the explanation.

I accept the comment.